docs-site/docs/user/project/projects/bot-accounts/types.md

```diff
@@ -4,4 +4,8 @@ id: types
 title: Bot Account Types
 ---
 
-<!-- TODO: Add description of the types -->
+The different bot account types are:
+
+- **Project bot** - A bot that is created within a project. This bot can only be associated to the project it was created in.
+- **Group bot** - A bot that is created within a group. It can also be added as a member to other global projects and groups.
```

Comment:
The Group Bot line can be removed as only project bot and project automation bot can be created for a project

Reply:
30ed197bcd870d4b31e3b30cf22454ef0bee829e